src/main/java/uk/gov/nationalarchives/pdi/step/jena/serializer/JenaSerializerStepMeta.java

```diff
@@ -215,7 +217,7 @@ public void getFields(final RowMetaInterface rowMeta, final String origin, final
 
 //        try {
 //            // add the target field to the output rows
-//            if (targetFieldName != null && !targetFieldName.isEmpty()) {
+//            if (isNotEmpty(targetFieldName)) {
```

Comment:
Can this try block be removed if it is not used?

Reply:
It can indeed.